src/stories/atoms/MovieCard.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "../assets/styles/MovieCard.scss";
+
+const MovieCard = ({ imageUrl, name, year, genreList }) => {
```

Comment:
Additionally, the component should receive a callback property to capture click event.

Reply:
`onClick={(e) => handleClick(e)` - if we just want to pass to callback all received from event data without changing/extending we can use next syntax - `onClick={handleClick)` and it saves us from extra-rerender (will discuss on Q&A)